docs/detections/rules-ui-create.asciidoc

```diff
@@ -261,37 +261,109 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 
 [discrete]
 [[create-esql-rule]]
-=== Create an ES|QL rule
+=== Create an {esql} rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
-. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
+Use {ref}/esql.html[{esql}] to query your source events and aggregate event data. Query results are returned in a table with rows and columns. Each row becomes an alert.
+
+To create an {esql} rule:
+
+. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page appears.
+. Select **{esql}**, then write a <<esql-rule-query-types,query>>. 
++
+TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help icon,20,20]) to open the in-product reference documentation for all {esql} commands and functions.
++
+[NOTE] 
```

Comment:
@vitaliidm let me know if this note needs any tweaks. Thanks in advance for the feedback!

Reply:
Thanks for the prompt feedback, @vitaliidm!

RE: Your first point about `max_signals` not being configurable through the UI:  

The ability to only configure `max_signals` through the API, and documenting this information there was sorta what I was alluding to [here](https://github.com/elastic/security-docs/pull/4062#discussion_r1374571790). It's also why I linked to the `max_signals` docs in the API docs, but it might be too presumptuous to assume that users would make the connection that the `max_signals` field is only configurable via the Create rule API. 

The clearest and most helpful way to doc this information is to put it where users need it, which is in the API docs. I still think the `max_signals` [docs](https://www.elastic.co/guide/en/security/master/rules-api-create.html#opt-fields-all) is the appropriate place to put this information, even though the description for that field applies generally to all rule types. Adding this information as a note shows that there's a caveat to the functionality that users should be aware of when they're working with a specific rule.

Reply:
@vitaliidm I'll merge this PR and let's continue this convo over in https://github.com/elastic/security-docs/issues/4195.